src/System.Device.Gpio/System/Device/Gpio/Drivers/UnixDriver.Linux.cs

```diff
@@ -14,7 +14,7 @@ public static UnixDriver Create()
             {
                 driver = new LibGpiodDriver();
             }
-            catch (DllNotFoundException)
+            catch (PlatformNotSupportedException)
```

Comment:
Is it possible to have a static tryget factory method to return a driver and avoid the exception handling entirely?

Reply:
Seems we don't need to catch DllNotFoundException anymore as https://github.com/dotnet/corefx/issues/32015 is implemented and we already have issue to fix this https://github.com/dotnet/iot/issues/157. So I am gonna address your comment with the #157